OpenXmlPowerTools/DocumentAssembler.cs

```diff
@@ -717,10 +717,18 @@ static object ContentReplacementTransform(XNode node, XElement data, TemplateErr
                                         XElement paragraph = tc.Elements(W.p).FirstOrDefault();
                                         XElement cellRun = paragraph.Elements(W.r).FirstOrDefault();
                                         string xPath = paragraph.Value;
+                                        bool cellIsOptional = false;
```

Comment:
This can be achieved more simply.

```
    XElement xElement = XElement.Parse(paragraph.Value);
    var xPath = (string)xElement.Attribute(PA.Select);
    var optionalString = (string)xElement.Attribute(PA.Optional);

    try
    {
        newValue = EvaluateXPathToString(d, xPath, optionalString?.ToLower() == "true");
    }
```